### PR TITLE
[REVIEW] Removing GIS functionality

### LINKED ISSUE #2937: [FEA] Remove GIS functionality
GIS functionality has moved to the cuSpatial library. Remove cudf/gis.hpp and associated implementation files / cmake / cython.

### PR DESCRIPTION
**Pull request by rgsl888prabhu**
close #2937

### CODE REVIEW

**Comment by rgsl888prabhu** [comment]
rerun tests

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 - PR #2904 Move gpu decompressors to cudf::io namespace
 - PR #2965 Fix slow orc reader perf with large uncompressed blocks
+- PR #2988 Removing GIS fucntionality
```
Comment (harrism) [inline CHANGELOG.md]:
```suggestion
- PR #2988 Removing GIS functionality (now part of cuSpatial library)
```

**Comment by harrism** [comment]
rerun tests

**Comment by kkraus14** [comment]
@felipeblazing @williamBlazing is this okay on your end?

**Comment by harrism** [comment]
@rgsl888prabhu please resolve conflicts so we can merge. Thanks!

**Comment by rgsl888prabhu** [comment]
@harrism @kkraus14 conflicts have been resolved

**Comment by rgsl888prabhu** [comment]
rerun tests

**Comment by rgsl888prabhu** [comment]
rerun tests

**Comment by rgsl888prabhu** [comment]
rerun tests

**Comment by rgsl888prabhu** [comment]
rerun tests

**Comment by harrism** [comment]
@rgsl888prabhu just need to merge branch-0.11 into yours to resolve conflicts and we can merge. Sorry for inconvenience, caused by merging #3029

**Comment by harrism** [comment]
@rgsl888prabhu I noticed that `conda/recipes/libcudf/meta.yaml` still contains a reference to gis.hpp, can you please remove it?

**Comment by harrism** [comment]
Ignore previous comment, I see it's done.

**Comment by rgsl888prabhu** [comment]
For a while I thought someone added a legacy path for it :)

**Comment by rgsl888prabhu** [comment]
> @felipeblazing @williamBlazing is this okay on your end?

@felipeblazing @williamBlazing Shall we proceed with the GIS removal ?

**Comment by williamBlazing** [comment]
> > @felipeblazing @williamBlazing is this okay on your end?
> 
> @felipeblazing @williamBlazing Shall we proceed with the GIS removal ?

Yes, its ok with us to remove this

**Pull request merged**